src/components/common.js

```diff
@@ -92,7 +92,7 @@ export const Link = Utils.forwardRefWithName('Link', ({ disabled, variant, child
     style: {
       color: disabled ? colors.dark(0.7) : colors.accent(variant === 'light' ? 0.3 : 1),
       cursor: disabled ? 'not-allowed' : 'pointer',
-      fontWeight: 500, display: 'inline'
+      fontWeight: _.getOr(500, 'style.fontWeight', props), display: 'inline'
```

Comment:
As discussed, we hope the merge takes care of this, but if not here is another option:
```suggestion
      fontWeight: props.style?.fontWeight || 500, display: 'inline'
```